src/rgw/driver/rados/rgw_sal_rados.cc

```diff
@@ -1296,11 +1421,34 @@ int RadosBucket::commit_logging_object(const std::string& obj_name,
   }
 
   const auto temp_obj_name = to_temp_object_name(this, obj_name);
+  rgw_pool temp_data_pool;
+  bool is_ec_pool = false;
+
+  ret = get_logging_temp_object_pool (dpp, this, store, obj_name_oid, temp_data_pool, is_ec_pool, y);
```

Comment:
we can probably optimize the calls to `get_committed_logging_object()` and `get_logging_temp_object_pool()`.
* both are getting attributes fro the object_name object. 
* both create an io context based on the data_pool